### ISSUE TITLE
Played Graveyard beatmaps search

### ISSUE THREAD

**Issue by jchy145** [ppy/osu-web#10190]
![image](https://github.com/ppy/osu-web/assets/133388823/e2cc7d87-905c-4260-9094-808e763ef895)

Played Graveyarded maps from "Historical Plays" section on the profile do not get fetched to the beatmap listings page.

**Comment by nanaya** [comment]
I don't think it can be done from osu-web as there's no record of such plays in the database (after 24h, that is).
